src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+                        collaborative atmosphere that makes startup work so meaningful. 
+                        The judges asked sharp and insightful questions, and the organizers 
+                        of the event made every transition so smooth. Even though we felt 
+                        the adrenaline of presenting first of all the teams, the excitement 
+                        never left us for the entire competition as we were hooked by the 
+                        amazing ideas from all the other teams and the fantastic fireside 
+                        chat with Dennis Woodside. Announcing the results live at the end of 
+                        the competition was a thriller, and it was surreal when ReneCycle
+                        was announced first. I feel proud to have participated in such a 
+                        great event and feel immense gratitude towards all the organizers, 
+                        mentors, judges, and participants who made this event possible. 
+                        A huge shoutout to our PCV assigned mentor Nicholas Yiu for giving 
+                        us such valuable insights." -- Henry</Text>
+                </ColumnText>
+                <ColumnText css={`padding-left: 2vw`}>
+                    <b><Subtitle fontSize="0.6rem">Future Plans?</Subtitle></b>
+                    <Text>"ReneCycle is looking forward to using the funding from PCV to 
+                        invest further into the product development of our smart bike lock! 
+                        We are looking into partnering with new corporate campuses and 
+                        municipal districts this Summer, and will be posting new updates on 
+                        our website, LinkedIn, and instagram accounts. We've had an exciting 
+                        product launch at Swarthmore College this Spring and will be soon 
+                        providing new content on bike usage, product resilience, and footage 
+                        of real student experiences. Stay tuned for exciting content and 
+                        insights from the ReneCycle team!" -- Mina<br/>
+                        <br/>
+                        <b>LinkedIn:</b><br/>
+                        <Link href="https://www.linkedin.com/company/renecycle">https://www.linkedin.com/company/renecycle </Link><br/>
+                        <b>Website:</b><br/>
+                        <Link href="https://www.renecycle.com/">https://www.renecycle.com/</Link><br/>
+                        <b>Instagram:</b><br/>
+                        <Link href="https://www.instagram.com/renecycle.inc/">https://www.instagram.com/renecycle.inc/</Link>
+                    </Text>
+                </ColumnText>
+            </Columns>       
+        </div>
+    )
+}
+
+const SecondPlace = () => {
+    return (
+        <div>
+            <Subtitle css={`margin-top: 2vw`}>Second Place - Folia</Subtitle>
```

Comment:
Every single Subtitle in each component here has the same style. Extract out a SpacedSubtitle = s(Subtitle) above to follow DRY